### PR TITLE
Add configuration/environment hostname errors on the status page

### PR DESCRIPTION
**Pull request by remicalixte**
### What does this PR do?

Add configuration/environment hostname errors on the status page

### Motivation

Since #2103 we display hostname errors on the status page

### CODE REVIEW

**Comment by remicalixte** [comment]
[Update the status page to display hostname errors and provider](https://trello.com/c/DDTRZyoY/932-update-the-status-page-to-display-hostname-errors-and-provider)

File: pkg/util/hostname.go [hunk 1]
```diff
@@ -118,6 +118,10 @@ func GetHostname() (string, error) {
 		return name, err
 	}
 
+	expErr := new(expvar.String)
+	expErr.Set(err.Error())
+	hostnameErrors.Set("configuration/environment", expErr)
```
Comment (hush-hush) [inline pkg/util/hostname.go]: Could we add a error log line 56 when a hostname is not valid ?

**Review by hush-hush — COMMENTED** [review]
On small comment, feel free to merge after that.

**Review by truthbk — APPROVED** [review]
We might get some redundant logging, but that's fine. Merging as you addressed @hush-hush comments. Thanks!

**Pull request merged**